charts/linkerd2/templates/controller.yaml

```diff
@@ -128,7 +128,10 @@ spec:
       - {{- include "partials.proxy" $tree | indent 8 | trimPrefix (repeat 7 " ") }}
       {{ if not .Values.cniEnabled -}}
       initContainers:
-      - {{- include "partials.proxy-init" . | indent 8 | trimPrefix (repeat 7 " ") }}
+      {{- if not (contains "443" ( .Values.proxyInit.ignoreOutboundPorts | toString ))}}
+      {{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" (printf "%s,443" ( .Values.proxyInit.ignoreOutboundPorts | toString )) -}}
+      {{- end}}      
```

Comment:
There are extra spaces at the end of this line that explain the diffs in the golden files. It seems the other templates except for identity.yaml do as well. Also, to avoid repetition, IMO it'd make sense to move this logic into `_proxy-init.tpl` itself.

Reply:
Please disregard my comment about moving this into `_proxy-init.tpl`. I realize now we only want this for the control-plane deployments... So the only thing left is to remove those spaces :wink:

Reply:
Done, @alpeb!